client/js/backstage.js

```diff
@@ -157,29 +157,29 @@ var BackstageView = Backbone.View.extend({
         this.$openButton    = this.$el.find('.silky-bs-open-button')
         this.$saveButton    = this.$el.find('.silky-bs-save-button')
         this.$saveAsButton  = this.$el.find('.silky-bs-save-as-button')
-		
+
 		this.main = new BackstagePlaces({ el: ".silky-bs-main", model: this.model })
 	},
 	activate : function() {
-	
+
 		this.$el.fadeIn(100)
-		
```

Comment:
i actually kinda like your text-editors behaviour, but i'll have to find a way to make all code conform to it. otherwise i'll commit stuff, your text-editor will fix it, and commit it back again, etc. etc. (and you'll have +138,385 -137,546 loc in no time! you will look very fickle!)
i think travis-ci (which integrates with github) might be able to help us out here.